File: Lesson 1/Lab1/Lab1a-Python/bank_account.py
class BankAccount:

    def __init__(self, balance_cdn, account_number, member_last_name):
        self.balance_cdn = balance_cdn
        self.account_number = account_number
        self.member_last_name = member_last_name

    def withdraw(self, amount_cdn):
        self.balance_cdn -= amount_cdn

    def deposit(self, amount_cdn):
        self.balance_cdn += amount_cdn

    def transfer(self, amount_cdn, recipient_account):
        recipient_account.deposit(amount_cdn)
        self.withdraw(amount_cdn)

File: Lesson 1 - test/lab2atest/src/BankAccount.java
class BankAccount {
    public double balanceCdn;
    public final String accountNumber;
    public final String memberLastName;

    BankAccount(final double balanceCdn, final String accountNumber, final String memberLastName) {
        this.balanceCdn     = balanceCdn;
        this.accountNumber  = accountNumber;
        this.memberLastName = memberLastName;
    }

    public void setBalanceCdn(final double balanceCdn)
    {
        this.balanceCdn = balanceCdn;
    }

    public double getBalanceCdn() {
        return balanceCdn;
    }

    public String getAccountNumber() {
        return accountNumber;
    }

    public String getMemberLastName() {
        return memberLastName;
    }

    public void withdraw(final double amountCdn) {
        balanceCdn -= amountCdn;
    }

    public void deposit(final double amountCdn) {
        balanceCdn += amountCdn;
    }

    public void transfer(double amountCdn, BankAccount recipientAccount) {
        recipientAccount.deposit(amountCdn);
        withdraw(amountCdn);
    }

}
